packages/admin-ui/src/Select/SelectPresenter.ts

```diff
@@ -0,0 +1,75 @@
+import { makeAutoObservable } from "mobx";
+import omit from "lodash/omit";
+import { SelectOptionsVm, SelectProps, SelectTriggerVm, SelectVm } from "./Select";
+import { SelectOption } from "./SelectOption";
+import { SelectOptionMapper } from "~/Select/SelectOptionMapper";
+
+interface ISelectPresenter<TProps extends SelectProps = SelectProps> {
+    vm: {
+        selectVm: SelectVm;
```

Comment:
Kinda on the fence whether the `Vm` suffix is needed when these already within the `vm` object. But doesn't matter probably in the grand scheme of things.

Reply:
Thank you, it makes sense 👍